src/lib.rs

```diff
@@ -32,8 +32,6 @@ mod error;
 mod key;
 mod protected;
 
-// Use pure Rust drop-in replacement `ethsign-crypto` of `parity-crypto`.
-#[cfg(feature = "ethsign-crypto")]
```

Comment:
This feature is redundant as ethsign-crypto is always used.
I even get error when trying to compile without default features as its Error is used without a feature gate.